src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -309,7 +300,6 @@ public function load(array $configs, ContainerBuilder $container)
                 'Symfony\\Component\\DependencyInjection\\Container',
 
                 'Symfony\\Component\\EventDispatcher\\Event',
-                'Symfony\\Component\\EventDispatcher\\ContainerAwareEventDispatcher',
```

Comment:
@Koc Actually, that would not have been an issue. This part of the code is never executed on PHP 7 (which is a requirement for Symfony 4).